trieTools/trieStatsPrinter/main.go

```diff
@@ -66,7 +71,7 @@ func startProcess(c *cli.Context) error {
 		return err
 	}
 
-	rootHash, err := hex.DecodeString(flagsConfig.HexRootHash)
+	rootHash, err := hex.DecodeString("66bb8f5603f95558fd96079d2316964f169ea5dad7aca5ce3ae7371b38c443ba")
```

Comment:
Testing artifact.

Reply:
Indeed, reverted the changes from this file as they were from a previous test.